.ci/push_dockerhub.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo "Pusing to dockerhub..."
```

Comment:
it's still in progress, that's the deployment step I haven started working on yet ;)